File: frontend/olympic_it_project/olympic_it_project/lib/dto/admin_manager/exam/anti_cheat_response.dart
class AntiCheatResponse {
  final int userId;
  final String fullName;
  final String type;
  final String createdAt;

  AntiCheatResponse({
    required this.userId,
    required this.fullName,
    required this.type,
    required this.createdAt,
  });

  factory AntiCheatResponse.fromJson(Map<String, dynamic> json) {
    return AntiCheatResponse(
      userId: json["userId"],
      fullName: json["fullName"],
      type: json["type"],
      createdAt: json["createdAt"],
    );
  }
}

File: backend/olympic_it_project/src/main/java/org/example/olympic_ot_project/dto/exam/anticheat/AntiCheatResponse.java
package org.example.olympic_ot_project.dto.exam.anticheat;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.example.olympic_ot_project.core.AntiCheatType;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AntiCheatResponse {
    private Integer userId;
    private String fullName;
    private AntiCheatType type;
    private LocalDateTime createdAt;
    private Integer violationCount;
    private boolean banned;
}